revelio/urls.py

```diff
@@ -19,6 +19,7 @@
 
 urlpatterns = [
     path('', include('authentication.urls')),
+    path('', include('ai.urls')),
```

Comment:
I'm quite concerned about this url, I think it would be the best practice to differentiate the root url between authentication.